### ISSUE TITLE
Proposal: non-scaling property transformations

### ISSUE THREAD

**Issue by georeith** [paperjs/paper.js#999]
When you have a `Shape.Rectangle` and you set its `radius` property then you `scale` it the rounded corners transform with it. This is the same with stroke although it is possible to tell that not to scale.

If you set it's `size` property then the corners do not scale.

It would be great to be able to specify that corner-radius should be non scaling as right now the only way to scale a `Group` that contains a rounded rectangle and maintaining the corners is to iterate through all the children work out the new `x`, `y`, `width` and `height` and apply them directly.

Although having a non-scaling corner radius would be great, whats possibly more useful is being able to apply transformations directly to Shapes without scaling their other properties. As with strokes sometimes you don't want to scale it, other times you do (say you had zoom functionality).

**Comment by lehni** [comment]
This could be implemented with a `options` argument to `Item#transform()`. Please note that this is not the same as `strokeScaling` though, since those strokes don't even scale when you zoom the view, while the shape radius probably should scale then.

**Comment by georeith** [comment]
@lehni Yeah it's a tricky one because ellipse and circle need to scale radius (as that's their actual dimension) whereas radius is corner radius (which can affect dimension if the the radius in a given direction is more than half of its current size) when it comes to rectangles.

It makes sense to tell a rectangle not to scale its radius on a given transform but not an ellipse or a circle.

Indeed that's what I'm thinking that `strokeScaling` is perhaps too binary at the moment, sometimes you want it to scale, sometimes you don't.

So as far as I can see iterating through all the child shapes and setting size directly seems to do this. But doing that in transform is kind of bad as it doesn't respect `applyMatrix`, perhaps another function which does that and allows you to apply a transformation but directly to the item only on its dimensions. I imagine this would be a nightmare to do with matrices anyway you would have to inverse scale the properties like stroke right?
